src/pages/studyView/chartHeader/ChartHeader.tsx

```diff
@@ -602,6 +608,60 @@ export class ChartHeader extends React.Component<IChartHeaderProps, {}> {
             }
         }
 
+        if (this.props.chartType == ChartTypeEnum.MUTATION_DIAGRAM) {
+            items.push(
+                <li>
+                    <a
+                        className="dropdown-item"
+                        onClick={() =>
+                            this.props.store.onVisualizingMutationPlot(
+                                this.props.chartMeta.uniqueKey
+                            )
+                        }
+                    >
+                        <i
+                            className={classnames(
+                                'fa fa-xs fa-fw',
+                                'fa-eye',
+                                styles.menuItemIcon
+                            )}
+                            aria-hidden="true"
+                        />
+                        Visualize
```

Comment:
`Visualize` is not a great name, we need to be more specific. What we actually mean with visualize is to open the Results view page. We need to think about a better name.

Reply:
Agree. What about `explore`?